Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -619,6 +621,67 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
                         # Update Connector Counter
                         $connectorCounter += 1
                     }
+                    elseif ($objectKeyLowercase -eq "watchlists") {
+                        $watchlistData = $json.resources[0]
+                        #Handle CreateUiDefinition Base Step
+                        if ($watchlistCounter -eq 1) {
+                            $baseWatchlistStep = [PSCustomObject]@{
+                                name       = "watchlists";
+                                label      = "Watchlists";
+                                subLabel   = [PSCustomObject]@{
+                                    preValidation  = "Configure the watchlists";
+                                    postValidation = "Done";
+                                }
+                                bladeTitle = "Watchlists";
+                                elements   = @(
+                                    [PSCustomObject]@{
+                                        name    = "watchlists-text";
+                                        type    = "Microsoft.Common.TextBlock";
+                                        options = [PSCustomObject]@{
+                                            text = "Azure Sentinel watchlists enable the collection of data from external data sources for correlation with the events in your Azure Sentinel environment. Once created, you can use watchlists in your search, detection rules, threat hunting, and response playbooks. Watchlists are stored in your Azure Sentinel workspace as name-value pairs and are cached for optimal query performance and low latency. Once deployment is successful, the installed watchlists will be available in the Watchlists blade under 'My Watchlists'.";
+                                            link = [PSCustomObject]@{
+                                                label = "Learn more";
+                                                uri = "https://docs.microsoft.com/azure/sentinel/watchlists";
```

Comment:
Should we do an `aka.ms` link instead?

Reply:
Good call, will use https://aka.ms/sentinelwatchlists